lib/board.rb

```diff
@@ -3,6 +3,17 @@ def initialize
     @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
   end
 
+  WINNING_COMBOS = [
```

Comment:
Should the board know about how to win tic tac toe? (I can go either way on this)

Reply:
I was conflicted about this, but I made that decision because I felt it was the board's job to know what was on the board and where the pieces were.